frontend/lib/location/location_ffi.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/services.dart';
```

Comment:
What does `location_ffi` stand for?

Reply:
ffi stands for foreign function interface.